src/app/fdctl/config.c

```diff
@@ -537,6 +558,42 @@ init_workspaces( config_t * config ) {
     idx++;
   }
 
+  config->shmem.workspaces[ idx ].kind      = wksp_metrics;
+  config->shmem.workspaces[ idx ].name      = "metrics";
+  config->shmem.workspaces[ idx ].page_size = FD_SHMEM_GIGANTIC_PAGE_SZ;
+  config->shmem.workspaces[ idx ].num_pages = 1;
+  idx++;
+
+  config->shmem.workspaces[ idx ].kind      = wksp_metrics_quic;
+  config->shmem.workspaces[ idx ].name      = "metrics_quic";
+  config->shmem.workspaces[ idx ].page_size = FD_SHMEM_HUGE_PAGE_SZ;
+  config->shmem.workspaces[ idx ].num_pages = 1;
+  idx++;
+
+  config->shmem.workspaces[ idx ].kind      = wksp_metrics_verify;
```

Comment:
Curious why one workspace per tile -> metrics? My feeling is it might be cleaner with just one <everyone> -> metrics workspace

Reply:
We discussed this a few weeks ago and opted for one workspace per tile type for security.